types/better-sqlite3/index.d.ts

```diff
@@ -1,15 +1,14 @@
-// Type definitions for better-sqlite3 5.4
+// Type definitions for better-sqlite3 7.4
 // Project: http://github.com/JoshuaWise/better-sqlite3
 // Definitions by: Ben Davies <https://github.com/Morfent>
 //                 Mathew Rumsey <https://github.com/matrumz>
 //                 Santiago Aguilar <https://github.com/sant123>
 //                 Alessandro Vergani <https://github.com/loghorn>
 //                 Andrew Kaiser <https://github.com/andykais>
 //                 Mark Stewart <https://github.com/mrkstwrt>
+//                 Evan Hahn <https://github.com/EvanHahn>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
-
-import Integer = require("integer");
+// TypeScript Version: 4.3
```

Comment:
Is there any risk to doing this?

Reply:
I don't think so.

Reply:
this impacts interoperability across types, why this bump?

Reply:
> (Also, it has to be at least 3.2, because [that's when BigInts were added to TypeScript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html#bigint).)

that's OK, on DT 3.6 is the latest. I'd not bother with older versions

Reply:
Sounds good. I'll leave things as is, at 3.8. Let me know if that's wrong.